packages/core/helper-plugin/src/components/DynamicTable/index.js

```diff
@@ -140,17 +145,25 @@ const Table = ({
                       id: 'content-manager.components.TableDelete.label',
                       defaultMessage: '{number, plural, one {# entry} other {# entries}} selected',
                     },
-                    { number: entriesToDelete.length }
+                    { number: selectedEntries.length }
                   )}
                 </Typography>
-                <Button
-                  onClick={handleToggleConfirmDeleteAll}
-                  startIcon={<Trash />}
-                  size="L"
-                  variant="danger-light"
-                >
-                  {formatMessage({ id: 'global.delete', defaultMessage: 'Delete' })}
-                </Button>
+                {bulkActionsBar ? (
+                  React.cloneElement(bulkActionsBar, {
+                    ...bulkActionsBar.props,
+                    selectedEntries,
+                    clearSelectedEntries,
+                  })
```

Comment:
really interesting solution 👍 

We'll need to document the selectedEntries and clearSelectedEntries in the docs as otherwise people wouldn't know they're passed

Reply:
Have we considered going the extra mile and instead of passing components as props to use as slots refactoring this to be composable? 🤔 I'm not a fan of `cloneElement`. What happens if we want to extend this toolbar of the table even more? We're gonna end up with a bloated API imo.

If we're not going to refactor to be composable then consider some of the recommendations they make on the react website – https://react.dev/reference/react/cloneElement#alternatives

But if it were me, i'd make a context for this component and then use composability as it's unbelievably flexible.

Reply:
In order to move quick, our plan was to just pass the component as props for now and refactor the component to favor composability after we release the alpha. What are your thoughts on that approach? 

As far as not using cloneElement though, I think we could consider the alternatives you shared.

Reply:
Sure, I think for GA we should take the opportunity, but to get an alpha composability can be held back. Definitely lets take a look at removing `cloneElement` for the alpha though 👍🏼